src/DotVVM.Framework/Compilation/Javascript/Ast/JsExpressionInString.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace DotVVM.Framework.Compilation.Javascript.Ast
+{
+    public class JsExpressionInString : JsExpression
+    {
+        public string Expression { get; set; }
+
+        public JsExpressionInString(string expressions)
```

Comment:
It would be nicer to take ParametrizedString here.

I also think that we should handle operator precedence here.